ksp_plugin_adapter/flight_planner.cs

```diff
@@ -39,6 +39,14 @@ class FlightPlanner : WindowRenderer {
                               func       : RenderPlanner,
                               text       : "Flight plan",
                               options    : UnityEngine.GUILayout.MinWidth(500));
+      const float min_width_on_screen = 50;
+      const float min_height_on_screen = 50;
+      window_rectangle_.x = UnityEngine.Mathf.Clamp(window_rectangle_.x,
+                                                    min_width_on_screen - window_rectangle_.width,
```

Comment:
We use 80 character lines, please fold.
I'm also going ask you to put the all the `min_something` on the same side, I find that easier to understand:
```
      window_rectangle_.x =
          UnityEngine.Mathf.Clamp(
              window_rectangle_.x,
              -window_rectangle_.width + min_width_on_screen,
              UnityEngine.Screen.width - min_width_on_screen);
      window_rectangle_.y =
          UnityEngine.Mathf.Clamp(
              window_rectangle_.y,
              -window_rectangle_.height + min_height_on_screen,
              UnityEngine.Screen.height - min_height_on_screen);
```
should be fine.